plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointSchemaManager.java

```diff
@@ -53,19 +53,17 @@
             RowType.field("deletionTimestamp", BigintType.BIGINT),
             RowType.field("dataChange", BooleanType.BOOLEAN)));
 
-    private static final RowType PROTOCOL_ENTRY_TYPE = RowType.from(ImmutableList.of(
-            RowType.field("minReaderVersion", IntegerType.INTEGER),
-            RowType.field("minWriterVersion", IntegerType.INTEGER)));
-
     private final RowType metadataEntryType;
     private final RowType commitInfoEntryType;
 
```

Comment:
Remove empty line

Reply:
The empty line was intentional since the usage of `metadataEntryType` & `commitInfoEntryType` and `stringList` differ.